src/Windows/Avalonia.Win32/DxgiSwapchain/DXGI_MODE_DESC.cs

```diff
@@ -0,0 +1,13 @@
+﻿namespace Avalonia.Win32.DxgiSwapchain
+{
+    [System.Runtime.InteropServices.StructLayout(System.Runtime.InteropServices.LayoutKind.Sequential)]
```

Comment:
Sequential layout is automatically used for structs.

Reply:
Also, can we merge native struct/enum definitions to a single file? We usually do that for interop types.

Reply:
Removed, structs and enums have been consolidated into single cs files.